### PR TITLE
Gandlf recipe added

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
Hi, 

The license needs to be custom because this project is to be used for medical data and we want to have the "non-FDA clearance" message in the license.

**Comment by sarthakpati** [comment]
@conda-forge/help-python

I am getting the [following error for win-64](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=287530&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=106):

```
conda.exceptions.ResolvePackageNotFound: 
  - pytorch==1.7.1
```

Unsure how to resolve this. Could you please help?

**Comment by sarthakpati** [comment]
Tagging @conda-forge/help-python for merge

File: recipes/gandlf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+test:
+  imports:
+    - GANDLF
+    - GANDLF.models
+    - GANDLF.utils
+
+about:
+  home: "https://github.com/CBICA/GaNDLF"
+  license: BSD 3-Clause Modification
```
Comment (beckermr) [inline recipes/gandlf/meta.yaml:50]: Let's make this an SPDX license string.

Comment (beckermr) [reply recipes/gandlf/meta.yaml:50]: See the link above from the linter.

Comment (sarthakpati) [reply recipes/gandlf/meta.yaml:50]: Addressed.

File: recipes/gandlf/meta.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ test:
 
 about:
   home: "https://github.com/CBICA/GaNDLF"
-  license: BSD 3-Clause Modification
+  license: BSD-3-Clause
```
Comment (beckermr) [inline recipes/gandlf/meta.yaml:50]: Since it is modified you need to add some sort of extra thing here using AND and a custom license ref.

Comment (sarthakpati) [reply recipes/gandlf/meta.yaml:50]: Sorry, I don't follow. Could you please elaborate?

Comment (beckermr) [reply recipes/gandlf/meta.yaml:50]: Try something like

`BSD-3-Clause AND LicenseRef-Modified`

Comment (isuruf) [reply recipes/gandlf/meta.yaml:50]: That's misleading. Try

`LicenseRef-Cbica`

Comment (beckermr) [reply recipes/gandlf/meta.yaml:50]: Sure @isuruf. w/e is fine by me

Comment (sarthakpati) [reply recipes/gandlf/meta.yaml]: Added the following: `license: BSD-3-Clause AND LicenseRef-CBICA`

Is that okay?

Comment (isuruf) [reply recipes/gandlf/meta.yaml]: Nope. Please use `LicenseRef-CBICA` only

Comment (sarthakpati) [reply recipes/gandlf/meta.yaml]: Done.

**Pull request merged**